build/leafdoc-templates/html.hbs

```diff
@@ -116,6 +116,7 @@
 				<li><a href="#class">Class</a></li>
 				<li><a href="#evented">Evented</a></li>
 				<li><a href="#layer">Layer</a></li>
+				<li><a href="#interactive-later">Interactive layer</a></li>
```

Comment:
It's just the least ugly artifact I could think of - this way, classes show a section called "events inherited from interactive layer" and looks nice :-)